service/arm_templates/certificates.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "clusterID": {
+        "type": "string"
+      },
+      "keyVaultName": {
+        "type": "string"
+      },
+      "keyVaultSecretsObject": {
+        "type": "secureObject"
+      },
+      "keyVaultSKUFamily": {
+        "type": "string",
+        "defaultValue": "A"
+      },
+      "keyVaultSKUName": {
+        "type": "string",
+        "defaultValue": "standard"
+      }
+    },
+    "resources": [
+      {
+        "type": "Microsoft.KeyVault/vaults",
+        "name": "[parameters('keyVaultName')]",
+        "apiVersion": "2016-10-01",
+        "location": "[resourceGroup().location]",
+        "tags": {
+          "clusterID": "[parameters('clusterID')]"
+        },
+        "properties": {
+          "sku": {
+            "family": "[parameters('keyVaultSKUFamily')]",
+            "name": "[parameters('keyVaultSKUName')]"
+          },
+          "tenantId": "[subscription().tenantId]",
+          "accessPolicies": []
```

Comment:
The access policy will allow the cluster VMs to download the secrets files using a MSI (Managed Services Identify). This can't be added until the master nodes template has been merged.

Reply:
Do we need to track that or you'll remember?